program/rust/Cargo.toml

```diff
@@ -30,10 +32,9 @@ serde_json = "1.0"
 test-generator = "0.3.1"
 csv = "1.1"
 
-
 [features]
 debug = []
-library = []
+library = ["serde", "strum"]
```

Comment:
What is the reason for having two "subfeatures" serde and strum.
Instead of using `#[cfg(feature = "library")]` in the code

Reply:
Just for the flexibility, in reality most of the time someone is going to want both but this allows for this pattern:

```
pyth-oracle = { default-features = false, features = ["strum"] }
```

Etc.

Reply:
well thinking more, no Message will be exported without `library` feature though, right?

Maybe we can keep serde separate but i was thinking maybe we can hide strum from users entirely :?